Makefile

```diff
@@ -30,9 +30,13 @@ mainnet_build:
 	export REACT_APP_EXPLORER_SERVICE_BASE_URL=https://explorer-service.hathor.network/; \
 	npm run build
 
-.PHONY: mainnet_s3_sync
+.PHONY: mainnet_s3_sync clear_cloudfront_cache
```

Comment:
I think this is wrong.

Did it run?

Reply:
We should include `clear_cloudfront_cache` in the `mainnet_deploy`, not here

Reply:
Yes! My bad. changed